kyuubi-relocated-hive-metastore-client/pom.xml

```diff
@@ -36,6 +36,7 @@ under the License.
         <hadoop.version>3.3.6</hadoop.version>
         <httpclient.version>4.5.14</httpclient.version>
         <httpcore.version>4.4.16</httpcore.version>
+        <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
```

Comment:
change is not related.

note: this repo is supposed to be built and published via Java 8, not need to handle Java 11 or later cases

Reply:
Ohh, I didnt observed and used Java17, it worked fine with Java8. And I have taken care of this. Thanks for the review @pan3793